<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsTiledSceneElevationPropertiesWidgetBase</class>
 <widget class="QWidget" name="QgsTiledSceneElevationPropertiesWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tiled Scene Elevation Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mCrsGroupBox_2">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="title">
      <string>Elevation</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="syncGroup" stdset="0">
      <string notr="true">vectorgeneral</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QgsDoubleSpinBox" name="mOffsetZSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QgsDoubleSpinBox" name="mScaleZSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-99999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="mShiftZAxisButton">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Automatically sets the offset so that the lowest point in the scene sits at an elevation of 0.</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/images/themes/default/mActionRefresh.svg</normalon>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Elevation scaling and offset can be used to manually correct elevation values in the tiled scene at render time.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The scale is applied to the tiled scene elevation values before adding the offset.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCrsGroupBox_2</tabstop>
  <tabstop>mScaleZSpinBox</tabstop>
  <tabstop>mOffsetZSpinBox</tabstop>
  <tabstop>mShiftZAxisButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
